### PR TITLE
Update list spacing for latest from DM team

### PR DESCRIPTION
**Pull request by jimmynotjim**
Moved @scotchester and @nataliafitzgerald's work from https://github.com/cfpb/design-manual/issues/357 to this project.

## Additions

- Nested list spacing and style

## Removals

- Old list spacing

## Testing

- Checkout `update-lists` [branch in the sandbox](https://github.com/cfpb/capital-framework-sandbox/tree/update-lists).
- Run `npm run cf-link` here and in the sandbox.
- Run `npm run build && npm start` in the sandbox.
- Navigate to `http://localhost:3000/components/cf-core/cf-base.html`.

## Review

- @Scotchester 
- @nataliafitzgerald

## Screenshots

screen shot 2017-02-23 at 11 28 27 am

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-core/src/cf-base.less [hunk 1]
```diff
@@ -383,6 +383,11 @@ p + ol {
     margin-top: unit( -5px / @base-font-size-px, em );
 }
 
+ul ul,
+ol ul {
```
Comment (Scotchester) [inline src/cf-core/src/cf-base.less]: YOu forgot to add `ol ol` here. And, I suppose, `ul ol`.

**Comment by nataliafitzgerald** [comment]
@jimmynotjim @Scotchester 
When we have secondary bullets what is the space between the secondary bullet and the primary bullet that follows? In the screenshot above it appears that the space between the secondary bullet and the primary bullet that follows matches the space between paragraphs. Instead this space should match the space between primary bullets. This looks like it was correct in the [Codepen](http://codepen.io/Scotchester/pen/rrokQE?editors=1100) that @Scotchester created.

**Comment by Scotchester** [comment]
@nataliafitzgerald Hmm, it does look like that, @nataliafitzgerald. The CSS in the pen, which inherits from the live CSS on consumerfinance.gov, does do this correctly.

**Comment by jimmynotjim** [comment]
@nataliafitzgerald @Scotchester inspecting in the sandbox, it's 15px bottom `margin` it's inheriting in [this global list rule on L:371](https://github.com/cfpb/capital-framework/pull/450/files#diff-7049f34d37b91f0fc0ebe1c025293fdeR371)

File: src/cf-core/src/cf-base.less (hunk 1 above)
Comment (jimmynotjim) [inline src/cf-core/src/cf-base.less]: Whoops, thanks for catching that.

**Comment by Scotchester** [comment]
@jimmynotjim The question is why is it not getting the `:last-child` 'margin-bottom: 0;` rule? I'm guessing that's only in cfgov-refresh at this point.

**Comment by jimmynotjim** [comment]
@Scotchester I'm not seeing a `:last-child` rule anywhere in here other than the one for nav list items. I'm also not seeing it in the `unprosessed` files in cfgov-refresh. Do you think we should apply it to the [list here](https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-base.less#L364-L373)

**Comment by jimmynotjim** [comment]
Wait, I found it, a bit buried. https://github.com/cfpb/cfgov-refresh/blob/83c1281bd92868437c5a8b611d686f020d53f2dc/cfgov/unprocessed/css/enhancements/typography.less#L386-L390 It seems as though this should apply to the entire list I referenced above.

**Comment by Scotchester** [comment]
> It seems as though this should apply to the entire list I referenced above.

Agreed.

**Comment by jimmynotjim** [comment]
Fixed:

screen shot 2017-02-23 at 1 39 45 pm

**Comment by Scotchester** [comment]
Looks good, @jimmynotjim. I'd mention that update as a separate item in the change log, and then this is good to go.

**Comment by Scotchester** [comment]
Fixed `CHANGELOG.md` conflict. merging when Travis finishes.

**Pull request merged**